api/managementpb/backup/backups.proto

```diff
@@ -177,6 +177,27 @@ message RemoveScheduledBackupRequest {
 
 message RemoveScheduledBackupResponse {}
 
+message GetLogsRequest {
+  string artifact_id = 1 [
```

Comment:
probably this should be optional, and we can later add restore_id also.

Reply:
Please note that one artifact can have multiple jobs, for example PITR. What we should return in that case? Only last job logs, or all logs for this artifact?

Reply:
I wasn't aware of this.
Maybe just last job? Not sure

Reply:
Anyway, it's about implementation on managed side. So I resolve this conversation.